File: C/1589.c
#include <stdio.h>

main(){
    int i=0,num,raio1,raio2;
    
    scanf("%d",&num);
    
    for (i=0; i<num ; i++){
        scanf("%d",&raio1);
        scanf("%d",&raio2);
        printf("%d\n",(raio1+raio2));
    }

    return 0;
}

File: Ruby/1589.rb
i = 0
num = gets.chomp.to_i

while i < num
  i = i + 1
  raios =  gets.chomp
  raios = raios.split
  raio1 = raios[0].to_i
  raio2 = raios[1].to_i
  
  puts raio1 + raio2
end